Sources/BFKit/Apple/UIKit/WKWebView+Extensions.swift

```diff
@@ -26,21 +26,21 @@
 
 import Foundation
 import UIKit
+import WebKit
 
-// MARK: - UIWebView extension
+// MARK: - WKWebView extension
 
-/// This extesion adds some useful functions to UIWebView.
-public extension UIWebView {
-    // MARK: - Functions
-    
-    /// Load the requested website.
-    ///
-    /// - Parameter website: Website to load.
-    func loadWebsite(_ website: String) {
-        guard let url = URL(string: website) else {
-            return
-        }
-        
-        loadRequest(URLRequest(url: url))
+/// This exension adds some useful functions to WKWebView.
+extension WKWebView {
+  // MARK: - Functions
+  
+  /// Load the requested website.
+  ///
+  /// - Parameter website: Website to load
+  func load(_ website: String) {
```

Comment:
Sorry for the misunderstanding, I meant to change `Website` to `website`, so the right signature would be `load(webiste website: String)`

Reply:
@FabrizioBrancati Ah like that let me change that for you xD

Reply:
@FabrizioBrancati Actually when I change it to:
`load(website website: String)` 
Swift throws a warning: Extraneous duplicate parameter name; 'website' already has an argument label

That's why I made it:
`load(Website website: String)`

Reply:
Oops sorry, the right signature would be `load(website: String)`, because with that the parameter will be the same of the signature.